pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -3045,6 +3045,10 @@ protected CompletableFuture<Response> internalExamineMessageAsync(String initial
                     try {
                         PersistentTopic persistentTopic = (PersistentTopic) topic;
                         long totalMessage = persistentTopic.getNumberOfEntries();
+                        if(totalMessage <= 0) {
+                            throw new RestException(Status.INTERNAL_SERVER_ERROR,
```

Comment:
i think we can change http status code to 422 Unprocessable Content or 412 Precondition Failed

Reply:
Great. 412 accepted

Reply:
@lifepuzzlefun PTAL

Reply:
@Technoboy- @codelipenghui @lordcheng10  @HQebupt PTAL